### ISSUE TITLE
Error running the application

### ISSUE THREAD

**Issue by leekaimun** [TheGoldenEye/polka-store#3]
The program simply exists upon startup. I used the template config. The log file showed:
```
2021-07-31 20:34:46        REGISTRY: Error: Unable to create Enum via index 13, in V0, V1, V2, V3, V4, V5, V6, V7, V8, V9, V10, V11, V12
    at assert (/home/lwr/projects/polka-store/node_modules/@polkadot/util/assert.cjs:34:11)
    at createFromValue (/home/lwr/projects/polka-store/node_modules/@polkadot/types/codec/Enum.cjs:87:20)
    at decodeFromValue (/home/lwr/projects/polka-store/node_modules/@polkadot/types/codec/Enum.cjs:118:14)
    at decodeEnum (/home/lwr/projects/polka-store/node_modules/@polkadot/types/codec/Enum.cjs:142:10)
    at new Enum (/home/lwr/projects/polka-store/node_modules/@polkadot/types/codec/Enum.cjs:196:21)
    at new <anonymous> (/home/lwr/projects/polka-store/node_modules/@polkadot/types/codec/Enum.cjs:211:9)
    at new Type (/home/lwr/projects/polka-store/node_modules/@polkadot/types/create/registry.cjs:349:16)
    at decodeU8a (/home/lwr/projects/polka-store/node_modules/@polkadot/types/codec/utils/decodeU8a.cjs:24:19)
    at decodeStruct (/home/lwr/projects/polka-store/node_modules/@polkadot/types/codec/Struct.cjs:98:41)
    at new Struct (/home/lwr/projects/polka-store/node_modules/@polkadot/types/codec/Struct.cjs:132:26)
2021-07-31 20:34:46        RPC-CORE: getMetadata(at?: BlockHash): Metadata:: createType(Metadata):: Unable to create Enum via index 13, in V0, V1, V2, V3, V4, V5, V6, V7, V8, V9, V10, V11, V12
2021-07-31 20:34:46        API/INIT: Error: FATAL: Unable to initialize the API: createType(Metadata):: Unable to create Enum via index 13, in V0, V1, V2, V3, V4, V5, V6, V7, V8, V9, V10, V11, V12
    at EventEmitter.value (/home/lwr/projects/polka-store/node_modules/@polkadot/api/base/Init.cjs:91:25)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
2021-07-31 20:34:46        API/INIT: Error: createType(Metadata):: Unable to create Enum via index 13, in V0, V1, V2, V3, V4, V5, V6, V7, V8, V9, V10, V11, V12
    at createTypeUnsafe (/home/lwr/projects/polka-store/node_modules/@polkadot/types/create/createType.cjs:66:11)
    at RpcCore._formatOutput (/home/lwr/projects/polka-store/node_modules/@polkadot/rpc-core/index.cjs:387:41)
    at callWithRegistry (/home/lwr/projects/polka-store/node_modules/@polkadot/rpc-core/index.cjs:253:42)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
2021-07-31 20:34:46        RPC-CORE: chain(): Text:: WebSocket is not connected
2021-07-31 20:34:46        RPC-CORE: name(): Text:: WebSocket is not connected
2021-07-31 20:34:46        RPC-CORE: version(): Text:: WebSocket is not connected
Error: WebSocket is not connected
    at assert (/home/lwr/projects/polka-store/node_modules/@polkadot/util/assert.cjs:34:11)
    at /home/lwr/projects/polka-store/node_modules/@polkadot/rpc-provider/ws/index.cjs:478:26
    at new Promise (<anonymous>)
    at WsProvider.send (/home/lwr/projects/polka-store/node_modules/@polkadot/rpc-provider/ws/index.cjs:476:12)
    at callWithRegistry (/home/lwr/projects/polka-store/node_modules/@polkadot/rpc-core/index.cjs:252:40)
    at Observable._subscribe (/home/lwr/projects/polka-store/node_modules/@polkadot/rpc-core/index.cjs:259:9)
    at Observable._trySubscribe (/home/lwr/projects/polka-store/node_modules/rxjs/internal/Observable.js:44:25)
    at Observable.subscribe (/home/lwr/projects/polka-store/node_modules/rxjs/internal/Observable.js:30:22)
    at Observable.ConnectableObservable.connect (/home/lwr/projects/polka-store/node_modules/rxjs/internal/observable/ConnectableObservable.js:47:18)
    at RefCountOperator.call (/home/lwr/projects/polka-store/node_modules/rxjs/internal/operators/refCount.js:33:49)
```

**Comment by LeSai** [comment]
Use the development branch it has updated dependencies and will fix the problems.

**Comment by TheGoldenEye** [comment]
The Polkadot API is constantly changing.
Please use the development branch with updated dependencies as noted by LeSei.

**Comment by TheGoldenEye** [comment]
I've just released a new release v1.6.0
